src/Renci.SshNet/Sftp/SftpFileStream.cs

```diff
@@ -166,6 +169,28 @@ public virtual byte[] Handle
         /// </value>
         public TimeSpan Timeout { get; set; }
 
+        private SftpFileStream(ISftpSession session, string path, FileAccess access, int bufferSize, byte[] handle, long position)
+        {
+            Timeout = TimeSpan.FromSeconds(30);
```

Comment:
This 30 seconds timeout should be mentioned in documentation.

Reply:
Agree, but this is my "blame" only as a result of refactoring. The whole goal of my PR is to support async and stop using timeouts.